provisioning/service/tests/Config/IndividualEnrollmentTests.cs

```diff
@@ -74,7 +76,10 @@ public class IndividualEnrollmentTests
             "   \"provisioningStatus\":\"" + SampleProvisioningStatus + "\",\n" +
             "   \"createdDateTimeUtc\": \"" + SampleCreateDateTimeUTCString + "\",\n" +
             "   \"lastUpdatedDateTimeUtc\": \"" + SampleLastUpdatedDateTimeUTCString + "\",\n" +
-            "   \"etag\": \"" + SampleEtag + "\"\n" +
+            "   \"etag\": \"" + SampleEtag + "\",\n" +
```

Comment:
I would rather leave this as is (to check that if the Capabilities is missing, we will not send it over the wire) and create a different UT.
String concatenation should work for the expected JSon.

Reply:
We could have 2 UTs for this - when set to true and when set to false.